<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlyWindow</class>
 <widget class="QMainWindow" name="OnlyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>421</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>421</width>
    <height>191</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сформировать Пароль</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="idxLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Идентификатор</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idxLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="qLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QLabel" name="qValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QPushButton" name="genPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сформировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
